lib/modules/datasource/npm/get.ts

```diff
@@ -67,6 +68,134 @@ const PackageSource = z
   ])
   .catch({ sourceUrl: null, sourceDirectory: null });
 
+const DepResponse = z
+  .object({
+    versions: LooseRecord(
+      z.object({
+        repository: PackageSource,
+        homepage: z.string().nullish().catch(null),
+        deprecated: z.string().nullish().catch('Unknown deprecation reason'),
+        gitHead: z.string().nullish().catch(null),
+        dependencies: LooseRecord(z.string()).nullish().catch(null),
+        devDependencies: LooseRecord(z.string()).nullish().catch(null),
```

Comment:
what about peer dependency, engines and package manager? especially the node versions are helpful for further filtering

Reply:
I know nothing about these fields, I'm just passing them to the result, as it was before. It's possible we actually don't use them anywhere.